project_task_code/views/project_view.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="project_task_code_form_view" model="ir.ui.view">
+        <field name="name">project.task.code.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field name="code" class="oe_inline" />
+                <span class="oe_inline"> - </span>
+            </field>
+        </field>
+    </record>
+    <record id="project_task_code_tree_view" model="ir.ui.view">
+        <field name="name">project.task.code.tree</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_tree2" />
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field name="code" />
+            </field>
+        </field>
+    </record>
+    <record id="project_task_code_kanban_view" model="ir.ui.view">
+        <field name="name">project.task.code.kanban</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_kanban" />
+        <field name="arch" type="xml">
+            <field name="name" position="before">
```

Comment:
Field "name" exists twice in the kanban view, once for non-active tasks (shows with strikethrough), once for active tasks. This field selector only targets the first one, so you only see the code field on archived tasks.

Reply:
I tried fixing it but it does it actually does not interact well with the `name_with_subtask_count` widget : 

- there is no space between code and name fields
- only the code in stroke trough when the task is archived.

<img width="253" alt="Screenshot 2023-10-26 at 16 38 52" src="https://github.com/OCA/project/assets/3082119/0601db30-e922-4f64-a79c-0f557e1a93b1">
<img width="258" alt="Screenshot 2023-10-26 at 16 39 17" src="https://github.com/OCA/project/assets/3082119/8e8ee0ee-5e76-42cc-8a24-b887d00bb34f">

Reply:
This produces
```xml
<xpath expr="//field[@name='name']/.." position="before">
    <field name="code" class="me-2"/>
</xpath>
```

<img width="244" alt="Screenshot 2023-10-26 at 17 17 52" src="https://github.com/OCA/project/assets/3082119/784f6524-37ff-4195-aad8-3ef9521162df">

For both archived and unarchived tasks. The `s` tag is overiden by the `name_with_subtask_count` widget.

Reply:
Done